srcs/commands/Join.cpp

```diff
@@ -6,10 +6,13 @@ static std::string getChannelName(std::string channel)
 	std::size_t found = channel.find('#');
```

Comment:
We also have a channel class getter that iss named the same, can we use it, or name this one differently?